src/store/thunks/posts.js

```diff
@@ -7,7 +7,9 @@ export const fetchPosts = () => (dispatch) => {
     .then((snapshot) => {
       const postArray = [];
       snapshot.forEach((post) => {
-        postArray.push(post.data());
+        const obj = post.data();
```

Comment:
Again, `obj` can be named better to represents what the variable contains

Reply:
done